splunk_add_on_ucc_framework/global_config_update.py

```diff
@@ -316,3 +325,26 @@ def _stop_build_on_placeholder_usage(
                     exc_msg % ("input service", service["name"])
                 )
     global_config.update_schema_version("0.0.8")
+
+
+def _dump_enable_from_global_config(
+    global_config: global_config_lib.GlobalConfig,
+) -> None:
+    if global_config.has_inputs():
+        exc_msg = "`enable` attribute found in input's page table action."
+
+        # Fetch the table object from global_config
+        table = (
+            global_config.content.get("pages", {}).get("inputs", {}).get("table", {})
+        )
```

Comment:
table can also be in each service 
https://splunk.github.io/addonfactory-ucc-generator/inputs/tabs/

Reply:
updated the logic for the same